Resources/lu-LaboratoryTestResult-DiagnosticReport.xml

```diff
@@ -46,6 +46,40 @@
         <comment value="LaboratoryTestResult" />
       </mapping>
     </element>
+    <element id="DiagnosticReport.extension">
+      <path value="DiagnosticReport.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+      <min value="1" />
+    </element>
+    <element id="DiagnosticReport.extension:DiagnosticReportCompositionR5">
+      <path value="DiagnosticReport.extension" />
+      <sliceName value="DiagnosticReportCompositionR5" />

```

Comment:
Should the warnings be removed? Change the slice names to start with lower case letter.

Reply:
I checked this and the sliceName had to be altered to be camelCase to be in line with the profiling guidelines.